lib/matplotlib/cm.py

```diff
@@ -18,6 +18,7 @@
 import matplotlib.cbook as cbook
 from matplotlib._cm import datad
 from matplotlib._cm import cubehelix
+from matplotlib._cm_listed import cmaps as cmaps_listed
```

Comment:
So cmaps created with this method will not have reversed variants like `viridis_r`?